extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -6,7 +6,7 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 	'wikia.document',
 	'wikia.log',
 	'wikia.window'
-], function (googleSlots, helper, doc, log, window) {
+], function (googleSlots, recovery, doc, log, window) {
```

Comment:
I talked with @fraszczakszymon offline. We decided to work on this step by step and extract the part no needed by Mercury to different packages. Don't worry about my comment above right now, however it's good to keep that in mind